perllib/FixMyStreet/App/Controller/Report.pm

```diff
@@ -590,6 +588,7 @@ sub inspect : Private {
                     confirmed => $timestamp,
                     user => $c->user->obj,
                     photo => $c->stash->{upload_fileid} || undef,
+                    problem_state => $problem->state,
```

Comment:
Feels a bit odd to have it set update_params for the state change to decide whether to add this comment that then will set it again anyway, but it's as good an indicator as anything else I guess.

Reply:
Agreed, couldn't think of a simpler way than what we already had.